field_rrule/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Repetition rules
+================
+
+This module was written to offer a field type that holds rrules according to RFC 2445.
```

Comment:
Hmm, I read that [RFC2445](https://tools.ietf.org/html/rfc2445) is obsoleted by [RFC5545](https://tools.ietf.org/html/rfc5545)...